practice_2/AnimalsApp/AnimalsListModule/View/Cells/BottomCell/AnimalsBottomCell.swift

```diff
@@ -0,0 +1,52 @@
+import UIKit
+
+class AnimalsBottomCell: UICollectionViewCell {
+    static let identifier = "animalsBottomCell"
+    static let quantityOfAnimals = 3
+
+    var navigationController: UINavigationController?
+    var animals = [Animal]()
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(UINib(nibName: "AnimalsBottomCVCell", bundle: nil),
+                                forCellWithReuseIdentifier: AnimalsBottomCVCell.identifier)
+    }
+}
+
+// MARK: UICollectionViewDataSource
+extension AnimalsBottomCell: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        animals.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView,
+                        cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(
+                withReuseIdentifier: AnimalsBottomCVCell.identifier,
+                for: indexPath) as? AnimalsBottomCVCell else { fatalError("Can`t create the cell") }
```

Comment:
If cell can not be dequeued it can be created from scratch.
Using of fatalError is not a good practice.

Reply:
Thanks for the comment. I follow you that "fatalError" is not good practice.
What is scratch?
Perhaps you could show a good example of how to handle this. Google didn't help me solve this issue.
Thanks in advance.

Reply:
Answered in our school channel.
Good question though.

Reply:
Thanks. Tried to fix.